docs/ops/normalization/LRN_1.md

```diff
@@ -44,19 +44,21 @@
 
 * **1**: `data` - input tensor of any floating point type and arbitrary shape. Required.
 
-* **2**: `axes` - specifies indices of dimensions in `data` that define normalization slices. Required.
+* **2**: `axes` - specifies indices of dimensions in `data` that define normalization slices. There are two sets of values supported: axes = {1} (that corresponds to across channel mode) and axes = {2, 3, ..., len(data.shape) - 1} (corresponds to within channel mode). Required.
```

Comment:
Please, remove the added note about supported values. The specification is a generic description of the operation and the fact that some plugins does not comply to it does not mean that we need to state plugins limitation in the specification.

Reply:
It's not just some plugins - it's all plugins. I think it's beneficial to the user to specify the limitations here, instead of him discovering them by trial and error.

Reply:
I understand this, but we don't write about the limitations in the documentation. Please, remove it.